app/components/destination_selector_component.html.erb

```diff
@@ -0,0 +1,24 @@
+<div data-scheduler-lookup-url='<%= paging_schedule_path(origin: form.object.paging_origin_library) %>'>
+  <% if pickup_destinations.one? %>
+    <div class='form-group'>
+      <div class='<%= helpers.label_column_class %> control-label'>
+        Will be delivered to
+      </div>
+      <div class='<%= helpers.content_column_class %> input-like-text'>
+        <%= destination_label(pickup_destinations.first) %>
+      </div>
+      <%= form.hidden_field :destination, value: pickup_destinations.first %>
+    </div>
+  <% elsif pickup_destinations.many? # Symphony %>
```

Comment:
```suggestion
  <% elsif pickup_destinations.many? %>
```

I'm not sure what this comment means? It seems like this scenario is possible with FOLIO too, isn't it?

Reply:
I didn't understand this comment either and I remarked upon that here: https://github.com/sul-dlss/sul-requests/pull/1638#discussion_r1291268738. In this case I'm just copying what was already there.

Reply:
Fixed in #1730